tree-sitter-darklang/grammar.js

```diff
@@ -290,6 +291,23 @@ module.exports = grammar({
     // Floats
     float_literal: $ => /[+-]?[0-9]+\.[0-9]+/,
 
+    //
+    // List
```

Comment:
let's add a TODO here of something like: "allow multi-line lists where a newline is 'interpreted' as a list delimiter (i.e. no `;` needed)"

Reply:
another TODO: support `[1; 2;]` (users ending with a `;` -- as part of being a 'tolerant' parser)